examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -1646,6 +1646,42 @@ otThreadSetMasterKey(
 }
 
 OTAPI 
+const uint8_t *
+OTCALL
+otThreadGetPSKc(
+    _In_ otInstance *aInstance 
+    )
+{
+    if (aInstance == nullptr) return nullptr;
+
+    uint8_t *Result = (uint8_t*)malloc(sizeof(otPSKc));
+    if (Result == nullptr) return nullptr;
```

Comment:
Please use code blocks for all if statements, to increase readability.
```
if (Result == nullptr)
{
    return nullptr;
}
```

Also, as pointed out below, you should only have one return statement at the end of the function.